libs/datasets/combined_datasets.py

```diff
@@ -114,6 +114,21 @@ def build_us_latest_with_all_fields() -> LatestValuesDataset:
     )
 
 
+def get_us_latest(state=None, fips=None) -> dict:
```

Comment:
why not have separate functions for state and fips? They are called from different places and have mostly separate code paths.

Reply:
yeah I think I agree with your thinking here.  I think I went with this because it returns the data in pretty much the same format.

Side note - we recently standardized all of the state level data to have a state fips code, we might be able to deprecate having to query this data by the state name - A lot of the modeling code handles getting state and county level data separately.